Modules/Challenges/test/Challenges.API.Tests.Integration/Models/ChallengeResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Newtonsoft.Json;
+
+namespace Challenges.API.Tests.Integration.Models;
+public class ChallengeResponse
+{
+    public Challenge? Result { get; set; }
+    [JsonIgnore]
```

Comment:
A success Challenge response doesn't include an Error property. If we don't ignore that, the json schema validation will fail.

Reply:
Are you sure? Because the property is marked as nullable. Doesn't the JSON schema validator recognize that?
If not, there is probably an attribute that can be used to mark it as optional.

Reply:
There's `[JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]` . The attribute is used to indicate that the JSON property should be ignored if its value is the default value for the data type.